nougat/evaluete.py

```diff
@@ -349,3 +349,84 @@ def computeGC(sequence):
     gcFraction = float(gcCount) / totalBaseCount
     return gcFraction
 
+
+def computeAssemblyStats(sample_config):
+
+    outfile = os.path.join("contig_stats", "contiguity.out")
+    if not os.path.exists("contig_stats"):
+        os.makedirs("contig_stats")
+
+    try:
+        sequence = sample_config["reference"]
+        minlength = sample_config["minCtgLength"]
+        genomesize = sample_config["genomeSize"]
+    except KeyError, e:
+        if e is "minCtgLength":
+            minlength = 1000
```

Comment:
That's a better solution, yes. I just didn't think of it at the time :+1: 
I wouldn't really need the `try` block, since I don't really plan to do any exception handling here at all